src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -50,117 +50,162 @@ class BenchmarkArgs:
+        execute_test_parser.add_argument("--benchmark-config", dest="benchmark_config",
+                                         help="absolute filepath to custom benchmark.ini config")
+        execute_test_parser.add_argument("--user-tag", dest="user_tag",
+                                         help="Attach arbitrary text to the meta-data of each metric record")
+        execute_test_parser.add_argument("--workload-params", dest="workload_params",
+                                         help="With this parameter you can inject variables into workloads. Parameters differs "
+                                         'for each workload type. e.g., --workload-params "number_of_replicas:1,number_of_shards:5"')
+        execute_test_parser.add_argument("--test-procedure", dest="test_procedure",
+                                         help="Defines a test procedure to use. You can find a list of test procedures by using "
+                                         'opensearch-benchmark list test-procedures. E.g. --test-procedure="ingest-only"')
+        execute_test_parser.add_argument("--exclude-tasks", dest="exclude_tasks",
+                                         help='Defines a comma-separated list of test procedure tasks not to run. E.g. --exclude-tasks="index-append"')
+        execute_test_parser.add_argument("--include-tasks", dest="include_tasks",
+                                         help="Defines a comma-separated list of test procedure tasks to run. By default, all tasks listed in a test procedure array are run."
+                                         ' E.g. --include-tasks="scroll"',)
+        execute_test_parser.add_argument("--capture-node-stat", dest="telemetry", action="append_const", const="node-stats",
+                                         help="Enable opensearch-benchmark to capture node stat metrics such as cpu, mem, jvm etc as well.")
+        execute_test_parser.add_argument("--capture-segment-replication-stat", dest="telemetry", action="append_const", const="segment-replication-stats",
+                                         help="Enable opensearch-benchmark to segment_replication stat metrics such as replication lag.")
+        execute_test_parser.add_argument("--telemetry-params", dest="telemetry_params",
+                                         help="Allows to set parameters for telemetry devices. Accepts json input.")
+        execute_test_parser.add_argument("-v", "--verbose", action="store_const", default=logging.INFO, const=logging.DEBUG, dest="logging_level",
+                                         help="Show more verbose output.")
+        execute_test_parser.add_argument("--ml-node-storage", dest="ml_node_storage",
+                                         help="User provided ml-node ebs block storage size defaults to 100Gb")
+
+        # command to run comparison
+        compare_parser = subparsers.add_parser("compare", help="Compare two IDs")
```

Comment:
add `--benchmark-config` arg for compare command as well, since we need to use the same config to fetch datastore details.